src/runtime_debug.js

```diff
@@ -22,24 +22,26 @@ function ignoredModuleProp(prop) {
   }
 }
 
-function unexportedMessage(sym, isFSSybol) {
-  var msg = "'" + sym + "' was not exported. add it to EXPORTED_RUNTIME_METHODS (see the FAQ)";
-  if (isFSSybol) {
-    msg += '. Alternatively, forcing filesystem support (-sFORCE_FILESYSTEM) can export this for you';
```

Comment:
Looks like we lose the specific filesystem integration here? Maybe not a big loss.

Reply:
Reinstated;